doc/UserGuide/CustomizingTheUPP/EnablingOutput.rst

```diff
@@ -0,0 +1,360 @@
+      ASIGO1( 5)=   0.2000
+      ASIGO1( 6)=   0.2500
+      ASIGO1( 7)=   0.3000
+      ASIGO1( 8)=   0.3500
+      ASIGO1( 9)=   0.4000
+      ASIGO1(10)=   0.4500
+      ASIGO1(11)=   0.5000
+      ASIGO1(12)=   0.5500
+      ASIGO1(13)=   0.6000
+      ASIGO1(14)=   0.6500
+      ASIGO1(15)=   0.7000
+      ASIGO1(16)=   0.7500
+      ASIGO1(17)=   0.8000
+      ASIGO1(18)=   0.8500
+      ASIGO1(19)=   0.9000
+      ASIGO1(20)=   0.9500
+      ASIGO1(21)=   1.0000
+
+.. _build-upp:
+
+
+
+Build/Compile the UPP
+----------------------
+
+After modifying the source code, users must (re)build/(re)compile the UPP to ensure that changes take effect: 
+
+.. code-block:: console
+
+   cd UPP/tests
+   ./compile_upp.sh
+
+This will generate the UPP executable in the ``UPP/exec`` directory. 
+
+.. _postprd:
+
+Prepare the post-processing working directory
+----------------------------------------------
+
+#. Create a post-processing output directory:
+
+   .. code-block:: console
+
+      cd $TOP_DIR
+      mkdir postprd
+
+   .. note::
+      This directory can be created anywhere, but default settings assume that is named ``postprd`` and created inside ``${TOP_DIR}``.
+
+#. Download the UPP utility script for running standalone UPP and change the permissions:
+
+   .. code-block:: console
+
+      cd postprd
+      wget https://raw.githubusercontent.com/wiki/NOAA-EMC/UPP/run_upp
+      chmod 755 run_upp
+
+#. Modifying the script
+
+   Users will need to edit directory paths and start date for the experiment. It may also be necessary to modify the run command, model type, and I/O file formats. For example:
+
+   .. code-block:: console
+
+      ...
+      export TOP_DIR=/work2/noaa/epic/jsmith/hercules/test-sigma/
+      export POSTPRD_DIR=${TOP_DIR}/postprd
+      export UPP_HOME=${TOP_DIR}/UPP
+      export POSTEXEC=${UPP_HOME}/exec
+      export modelDataPath=${TOP_DIR}/control_p8_intel
+      export txtCntrlFile=${UPP_HOME}/parm/gfs/postxconfig-NT-gfs-f00-two.txt
+      export CRTMDIR=${UPP_HOME}/crtm/fix
+      # Set date/time information
+      export startdate=2021032206
+      export fhr=00
+      export lastfhr=06
+      export incrementhr=03
+      # Specify model ("GFS" or "LAM" in upper case)
+      export model="GFS"
+      # Set input format from model and ouput format from UPP
+      export inFormat="netcdfpara"
+      export outFormat="grib2"
+      # Set run command: 
+      # Single processor command example
+      # export RUN_COMMAND="${POSTEXEC}/upp.x "
+
+      #MPI sample command
+      # "-n 4" can be changed to a different number of tasks. 
+      export RUN_COMMAND="srun -A epic -n 4 ${POSTEXEC}/upp.x "
+
+      # The number of subdomains in the x-direction (set to >=2 for 2d decomposition)
+      export numx=1
+      ...
+
+   .. note::
+      The UPP expects ``atmf*``, ``sfcf*``, and ``GFSPRS*`` files for the GFS model, and it expects ``phyf*``, ``dynf*``, ``NATLEV*``, and ``PRSLEV*`` files for the LAM model. 
```

Comment:
@jdkublnick This note could probably be turned into a code snippet because I think it appears in the run ch now, too (or should).